x-pack/plugins/stack_alerts/public/alert_types/geo_containment/query_builder/__snapshots__/geo_containment_alert_type_expression.test.tsx.snap

```diff
@@ -27,10 +27,19 @@ exports[`should render BoundaryIndexExpression 1`] = `
               "clearCache": [MockFunction],
               "createField": [MockFunction],
               "createFieldList": [MockFunction],
+              "ensureDefaultDataView": [MockFunction],
```

Comment:
This snapshot change was required because I updated the `dataPluginMock` to include some more fields that are required for some Dashboard tests